services/reproducible-central/reproducible-central.service.js

```diff
@@ -0,0 +1,93 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object().pattern(Joi.string(), Joi.string()).required()
+
+export default class ReproducibleCentral extends BaseJsonService {
+  static category = 'dependencies'
+
+  static route = {
+    base: 'reproducible-central/artifact',
+    pattern: ':groupId/:artifactId/:version',
+  }
+
+  static openApi = {
+    '/reproducible-central/artifact/{groupId}/{artifactId}/{version}': {
+      get: {
+        summary: 'Reproducible Central Artifact',
```

Comment:
For the badge index/docs site, could we include a `description` property here with a short explanation (a couple of sentences) covering:

- Reproducible Central is a service that does X
- This badge shows Y

You can use markdown in the description property. This will help users to discover and understand the purpose of this badge.

Reply:
ok, I just added a description